worlds/alttp/Shops.py

```diff
@@ -278,6 +279,8 @@ def ShopSlotFill(multiworld):
                     if 'P' in multiworld.shop_shuffle[location.player]:
                         price_to_funny_price(multiworld, shop.inventory[location.shop_slot], location.player)
 
+    FillDisabledShopSlots(multiworld)
```

Comment:
this might require some explanation. is shop kind of broken in current main?

Reply:
It isn't, but it gets moved. FillDisabledShopSlots was originally in Main.py